jiralib.py

```diff
@@ -0,0 +1,65 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+from jira import JIRA
+
+from config import Config
+
+_ENV_VAR_NAMES = [
+    "JIRA_SERVER",
+    "JIRA_USERNAME",
+    "JIRA_TOKEN",
+    "JIRA_PROJECT",
+    "JIRA_PROJECT_ID",
+    "JIRA_COMPONENT",
+]
+
+
+def config():
+    c = Config(_ENV_VAR_NAMES)
+    c.Load()
+    return c
+
+
+class Jira:
+    def __init__(self, cfg):
+        self.project = cfg.vars["JIRA_PROJECT"]
+        self.project_id = cfg.vars["JIRA_PROJECT_ID"]
+        self.server = cfg.vars["JIRA_SERVER"]
+        self.component = cfg.vars["JIRA_COMPONENT"]
+
+        jiraOptions = {"server": self.server}
+        self.jira = JIRA(
+            options=jiraOptions,
+            basic_auth=(cfg.vars["JIRA_USERNAME"], cfg.vars["JIRA_TOKEN"]),
+        )
+
+    def issue_exists(self, issue):
+        query = f'project={self.project} AND text ~ "GITHUB:{issue.repo}-{issue.id}"'
+        issues = self.jira.search_issues(query)
+        return len(issues) != 0
+
+    def create_issue(self, issue):
+        issue_dict = dict()
```

Comment:
Please do not embed the type name in the variable name. You can call it `issue_data` for example.